### ISSUE TITLE
Blockaded monster/ship doesn't count as Stationary, impacts Creative Thought Network

### ISSUE THREAD

**Issue by Blue-Award** [freeorion/freeorion#4911]
Build b3de094, MP26, turn 198, 199... There's dimensional drifter blockaded on system Toranor. The system is within 200 uu of Collective Thought Network at Caramba.

The blockaded drifter can't actually move, but it doesn't count as Stationary, so is messing up the effect from Collective Thought Network.

Dunno, maybe it works as intented, but I would expect if a ship/monster didn't (couldn't) actually fly anywhere, it would not mess with collective thought network.

I appreciate it may not be obvious to test if ship has not moved other than through Stationary, so perhaps the Stationary test should consider that ship tried to move, but couldn't (for example due to being blockaded), so should still count as stationary?

As a side note, I guess this would also mess with something like Flanking policy, that the ships actually haven't moved, but are not considered Stationary?

**Comment by MichaelCourtney** [comment]
From memory there's a way to check for arrival turn?

**Comment by MichaelCourtney** [comment]
Yeah took a look stuff like dry dock is checking arrival turn, so if you wanted to fix it in scripting: Anything on a starlane is moving and anything that in system that arrived this turn is moving.

Though its probably better to fix the stationary check.
